lib/ts/axios.ts

```diff
@@ -188,6 +189,18 @@ export function responseInterceptor(axiosInstance: any) {
             const preRequestLSS = await getLocalSessionState(false);
             await saveTokensFromHeaders(response);
 
+            const frontToken = response.headers["front-token"];
```

Comment:
The front token is saved in multiple places, but I only see a single call to update the clock skew. It may be better if you moved this into `saveTokensFromHeaders` (for all interceptors, or into a place more centralized). We want to make sure we are not making a mistake by updating the front token but skipping the clock skew update.

Reply:
Good observation! I have moved it to `saveTokensFromHeaders` for all the interceptors.